app/models/holdings.rb

```diff
@@ -38,8 +38,9 @@ class Holdings
   # @param documents [Array<SolrDocument, Hash>] Solr documents or hash objects from a query
   # @param direction [String] The direction in which we're looking, either 'forward' or 'reverse'
   # @note Direction is used so that we don't add every shelf key to the items hash.
-  def initialize(documents:, direction:)
+  def initialize(documents:, direction:, classification:)
     @items = {}
+    @classification = classification
```

Comment:
If we add classification to the url params, does Holdings need it anymore? I'm asking because I genuinely don't know :)

Reply:
nope not needed after changing classification as a url param